src/lizaalert/courses/utils.py

```diff
@@ -23,3 +28,31 @@ class BreadcrumbLessonSerializer(serializers.Serializer):
 
     chapter_id = serializers.IntegerField()
     lesson_id = serializers.IntegerField()
+
+
+def check_finished_content(lesson_type=()):
```

Comment:
Опасно присваивать объекты в качестве дефолтных значений

```suggestion
def check_finished_content(lesson_type=None):
```